modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/main_view/props_transformer/utils/transformViewsItemProps.ts

```diff
@@ -6,14 +6,40 @@
 import {IView} from '@liferay/frontend-data-set-web';
 
 import dateFormat from '../../../common/utils/dateFormat';
+import formatActionURL from '../../../common/utils/formatActionURL';
 
+const OBJECT_ENTRY_FOLDER_CLASSNAME =
+	'com.liferay.object.model.ObjectEntryFolder';
+
+const getHrefLink = (item: any, props: any) => {
+	const actionId = 'actionLink';
```

Comment:
This is "copied" from SimpleActionLinkRenderer, which is the cellrenderer used in table view. Unfortunately for cards we cannot use it. 
The thing is with const `actionId = 'actionLink'`; Here I don't have the options available. And this is a fixed parameter added in https://github.com/liferay/liferay-portal/blob/master/modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/display/context/BaseSectionDisplayContext.java#L241 and https://github.com/liferay/liferay-portal/blob/master/modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/display/context/BaseSectionDisplayContext.java#L265